### DISCUSSION TITLE
Propuestas

### DISCUSSION THREAD

**Discussion by elpatronaco** [elpatronaco/pccomponentes-buy-bot#20] [Q&A]
Cosas que queráis ver nuevas, tiendas nuevas, cambios en lo que ya hay... 🥶🤙

**Comment by RodrigoTomeES** [comment]
@elpatronaco creo que en LDLC sería mejor comprobar el botón para mirar el stock, en plan si es añadir a la cesta, porque mira esta producto https://www.ldlc.com/es-es/ficha/PB00410146.html

**Reply by elpatronaco** [reply]
ciertoo, tengo que cambiar la forma de mirar stock de todas las tiendas realmente, porque en la mayoría busco el típico mensaje que pone "En stock", pero también puede estar en stock y poner otra cosa, o estar en otro idioma

**Reply by RodrigoTomeES** [reply]
En algunas tiendas usan una clase distinta para el botón de compra cuando está disponible y cuando no igual eso sirve

**Comment by DarthWendigo** [comment]
@elpatronaco quizás sería interesante cuando encuentra stock, saber de qué ha encontrado stock y a qué precio, ya sea igual o mayor al maxPrice fijado, puede servir para ver si quieres subir un poco ese maxPrice para próximas disponibilidades de stock. Actualmente cuando encuentra stock tienes que deducir de qué ha encontrado stock porque el resp no tiene info (al menos con coolmod). Ojo, no se si será posible pues de node yo regular jeje.

**Comment by RodrigoTomeES** [comment]
@elpatronaco no he podido ver lo de telegram todavía, pero si me he dado cuenta de que ahora no puedo ejecutar el bot en mi pc en las nuevas versiones porque se obliga a tener instalado Chromium como dependencia externa.

![imagen](https://user-images.githubusercontent.com/13077343/112326902-3d478a80-8cb5-11eb-8596-f33b1c65cdfa.png)

Creo que sería mejor que quien necesite añadir lo del executablePath (básicamente si ejecutas el bot en arm) lo añada al json.data, tu que opinas? He creado un pull request por si lo ves bien

**Comment by Sofian060** [comment]
Hola Muy Buenas Mi Propuesta es la siguiente el Bot funciona Bien pero al detectar que hay stock Se pone a petar telegram de mensajes no se si me entiendes. Hay alguna Manera de solucionar eso ??
